src/api.js

```diff
@@ -88,16 +93,21 @@ builder.declare({
     res.writeHead(200, {
       'Content-Type': 'text/event-stream',
       'Cache-Control': 'no-cache',
+      'Access-Control-Allow-Origin': '*',
```

Comment:
Do we need this line? `taskcluster-lib-api` seems to set this header [here](https://github.com/taskcluster/taskcluster-lib-api/blob/95c4d803c71411ba8f0976ee32ed544b89d1f745/src/api.js#L34).